bytecode/src/bytecode.rs

```diff
@@ -53,6 +53,7 @@ bitflags! {
         const HAS_DEFAULTS = 0x01;
         const HAS_KW_ONLY_DEFAULTS = 0x02;
         const HAS_ANNOTATIONS = 0x04;
+        const IS_CLASS = 0x08;
```

Comment:
Which flag in cpython does this correspond to?

See this list of flags:

https://github.com/python/cpython/blob/4a2edc34a405150d0b23ecfdcb401e7cf59f4650/Include/code.h#L71

It would be nice to keep this more or less similar.

Reply:
I think it's somewhat equivalent to `CO_NEWLOCALS`, I'll look into it.

Reply:
I'm still unaware how this issue is implemented in cpython